runtime/near-vm-runner/src/cache.rs

```diff
@@ -292,3 +292,61 @@ pub mod wasmer1_cache {
         }
     }
 }
+
+pub(crate) fn precompile_contract_impl(
+    vm_kind: VMKind,
+    wasm_code: &ContractCode,
+    config: &VMConfig,
+    cache: Option<&dyn CompiledContractCache>,
+) -> Result<bool, ContractPrecompilatonError> {
+    let cache = match cache {
+        None => return Ok(false),
```

Comment:
I think it's better to have this fn with `cache: &dyn CompiledContractCache`, there is nothing to do when cache is None

Reply:
My feeling is that this way it's somewhat easier to use.